content/en/docs/concepts/workloads/controllers/statefulset.md

```diff
@@ -295,6 +295,39 @@ a Pod is considered ready, see [Container Probes](/docs/concepts/workloads/pods/
 
 Please note that this field only works if you enable the `StatefulSetMinReadySeconds` [feature gate](/docs/reference/command-line-tools-reference/feature-gates/).
 
+#### PersistentVolumeClaim Policies
+
+{{< feature-state for_k8s_version="v1.23" state="alpha" >}}
+
+`.spec.persistentVolumeClaimRetentionPolicy` is an optional field that controls if and how PVCs are deleted during the lifecycle of a StatefulSet. There are two policies that are used.
+
+  - `whenDeleted` is applied when the StatefulSet is deleted.
+  - `whenScaled` is applied when the replica count of the StatefulSet is reduced, for example when scaling down the set.
+  
+These may each be set to either `Delete` or `Retain`.
+
+  - `Delete`. The PVCs created from the StatefulSet `volumeClaimTemplate` are deleted, for each Pod affected by the policy. With the `whenDeleted` policy, as all Pods are deleted when the Statefulset is deleted, all PVCs from the `volumeClaimTemplate` are deleted. With the `whenScaled` policy, only PVCs corresponding to Pod replicas being scaled down are deleted. Note PVC deletion only happens for this particular kind of Pod deletion. For example, Pods that are deleted due to being killed for rescheduling on a different node are not affected, and the PVCs will live to migrate to the new node.
+  - `Retain`. PVCs from the `volumeClaimTemplate` are not affected when their Pod is deleted. This is the behavior before this new feature.
+  
+The default for policies is `Retain`, matching the StatefulSet behavior before this new feature.
+
+Here is an example policy.
+
+```
+apiVersion: apps/v1
+kind: StatefulSet
+...
+spec:
+  persistentVolumeClaimRetentionPolicy:
+    whenDeleted: Retain
+    whenScaled: Delete
+...
+```
+
+The StatefulSet [controller](https://kubernetes.io/docs/concepts/architecture/controller/) adds owner references to PVCs, which are then deleted by the garbage collector. When the whenDeleted policy is Delete, an owner reference to the StatefulSet instance is placed on all PVCs. The whenScaled policy is a little more complicated, because it adds an owner reference to the PVC’s pod, and you usually don’t want all Pod deletions to remove their PVCs. This means that the StatefulSet controller must wait for scale-down events (when the StatefulSet replica count is changed to be less than the number of replicas currently in the cluster), and in that case add pod owner references only to the scaled down Pods.
```

Comment:
```suggestion
The StatefulSet {{<glossary_tooltip text="controller" term_id="controller">}} adds
[owner references](/docs/concepts/overview/working-with-objects/owners-dependents/#owner-references-in-object-specifications) to its PVCs, which are then deleted by the
{{<glossary_tooltip text="garbage collector" term_id="garbage-collection">}} at the appropriate time.
When you set the `whenDeleted` policy to `Delete`, an owner reference to the StatefulSet instance
is placed on all PVCs associated with that StatefulSet.

The `whenScaled` policy is a little more complicated, because it adds an owner reference to the
PVC’s pod, and you usually don’t want Pod deletions to remove the associated PVCs. This means that the
StatefulSet controller must wait for scale-down events (when the StatefulSet replica count is changed
to be less than the number of replicas currently in the cluster), and then sets pod owner references
only on the specific Pods that are due to be terminated due to the scale-down action.
```